build/web-client/tutorials/establishing-consensus.md

```diff
@@ -0,0 +1,105 @@
+# Establishing Consensus
+
+## Overview
+
+- Nimiq is a censorship-resistant, privacy-oriented, and decentralized blockchain-based payment protocol
+- A blockchain is a shared digital ledger that is kept up-to-date across all users in the network through a process called establishing consensus
+- Compared to most other blockchain systems, it allows a direct connection to the network using a light node
+- These light nodes use Zero Knowledge Proofs (ZKPs) to efficiently verify blockchain transactions and states
+
+## Learning
+
+Nimiq is a blockchain-based payment protocol. This technology offers several key benefits:
+
+- **Censorship-resistant:** No one can prevent you from using it
+- **Privacy-oriented:** You don’t need to provide any personal information to open an account
+- **Decentralized:** It’s not controlled by any single person but by all its users
+
+A blockchain is like a shared digital ledger. Keeping this ledger up to date across all users in the network is called establishing consensus. Traditionally, most blockchain systems require you to connect through a third party to avoid downloading the entire blockchain, which can be over 100 GB in size. This approach necessitates trusting the third party.
+
+Nimiq, however, allows direct connection to the network using a light node. Unlike full nodes, which download and verify the entire blockchain, light nodes only download a small portion of it. They rely on full nodes to provide the necessary information to verify transactions. This approach significantly reduces storage and computational requirements, making light nodes ideal for devices with limited resources, such as browsers or mobile devices.
```

Comment:
Although this guide is for a web client, which is a light node, both light and full nodes sync with ZKPs. I think it's worth mentioning that before continuing exclusively with the light node reference. Wdyt?